src/txn/txn_rollback_to_stable.c

```diff
@@ -1123,7 +1116,8 @@ __rollback_abort_fast_truncate(
   WT_SESSION_IMPL *session, WT_REF *ref, wt_timestamp_t rollback_timestamp)
 {
     /* Review deleted page saved to the ref. */
-    if (ref->page_del != NULL && rollback_timestamp < ref->page_del->durable_timestamp) {
+    if (ref->state == WT_REF_DELETED && ref->ft_info.del != NULL &&
```

Comment:
Can we assert ref->state == WT_REF_DELETED && ref->ft_info.del != NULL? I feel like that would be a nice check to have.

Reply:
No; empty pages can be in the `WT_REF_DELETED` state, without `ft_info.del` set.